Improve numerical readability scores of our README (#590)
UCL-ARC/python-tooling: README.md

</div>
<!-- markdownlint-restore -->

> This repository collects our recommendations for a research software project in Python.

Author: I got rid of `[UCL ARC]` because our name is already in the title up on line 4, and in the repo header in GitHub's UI.

> We have a [website] if you need some advice, and a template if you're starting from scratch.

Reviewer: This is vague, advice for _what_? And starting _what_ from stratch?

Author: > advice for what? 

Python tooling? 

> And starting what from stratch?

`a research software project in Python.` (the subject of the preceding sentence, pretty sure that's legal)

Author: ```
This repository collects our recommendations for a research software project in Python. # Unchanged
We have a [website] if you need some advice for what tools to use, and a template if you're starting a project from scratch.
```

This feels redundant (to me) but does improve the three remaining readability scores slightly. Is that OK?

> 🍪 Our template uses [cookiecutter] to set up a new package with our preferred tools ready to go.

Reviewer: ```suggestion
🍪 Our template uses the [cookiecutter] tool to set up a new package with our preferred tools ready to go.
```

> [!NOTE]

> > If you're making a package in a research community that already has a template, use theirs instead!

Reviewer: I think it's worth keeping an example here of a community that uses a specific/different template.

Author: This was a deliberate change.

The parentheses and abbreviation (`e.g.`) decrease readability. And I wasn't too enthusiastic about the specific example.

## How to use our template

> If you're comfortable with the command line, here are some quick start steps.

Reviewer: What if you're not comfortable with using the command line?
```suggestion
Here are some quick start steps. We assume you're already familiar with using the command line.
```

Author: Lol. Good point.

Author: Your suggestion reduces readability (although I agree that it's good). If we drop the second sentence, the scires are much better:

```
Here are some quick start steps.
```
That works, right?

We also have a more detailed [tutorial](./tutorial.md) that has been presented in workshops for researchers at UCL.

If you have [uv] installed, run:

```sh
uvx cookiecutter gh:ucl-arc/python-tooling --checkout latest
```

> If you don't want to use uv you can [install cookiecutter], then run without `uvx`:

Reviewer 2: > without `uvx`

Is this clear enough?

Author: The command to run is directly below... so ... yeeees?

I'll leave your comment here for the second reviewer to look at.

Reviewer: I think this is clear

```sh
cookiecutter gh:ucl-arc/python-tooling --checkout latest
```

> [Cookiecutter] will ask you questions to set up your project.

Author: Beginning of a sentence, though?

> [Cookiecutter] will ask you questions to set up your project.

Reviewer 2: I guess a tool name should always be the same. We're not talking about the object used to actually make cookies.

> [Cookiecutter] will ask you questions to set up your project.

Reviewer 2: I do agree with Vale here

> [Cookiecutter] will ask you questions to set up your project.

Author: Overruled. 🔨

Type your answer, or hit return without typing anything to use the defaults.

When finished, it prints some information to the terminal.
These are tips for next steps like creating a repository and making a website for your package.

It will have created a directory for your project:

```sh

> ls -ltr | tail -n1 # Show the last directory that was created

Reviewer 2: https://docs.astral.sh/ruff/rules/too-few-spaces-before-inline-comment/
```suggestion
ls -ltr | tail -n1  # Show the last directory that was created
```

> ls -ltr | tail -n1  # Show the last directory that was created

Reviewer: I find the number of non-exapanded flags in this command a bit overwhelming (but appreciate it's a hangover from before this PR, so doesn't need to be improved here)

Author: It looks like:
```
ls -lt --reverse | tail --lines=1
```
(`-l` and `-t` don't have expansions)

Is that better?

```

You can see the structure with the `tree` command.
In our example we've called our project `example-research-software-project`:

```sh
tree example-research-software-project
```

To start working on your project:

- change directory to your project directory,
- initialise a `git` repository,

> - and _install_ your new package editable mode.

Reviewer: Should this be an ordered (numbered) list, since it has to be done in the specific listed order?

Author: ```suggestion
1. change directory to your project directory,
2. initialise a `git` repository,
3. and _install_ your new package editable mode.
```
